build.gradle

```diff
@@ -474,6 +475,23 @@ project('spring-integration-cassandra') {
     }
 }
 
+project('spring-integration-cloudevents') {
+    description = 'Spring Integration CloudEvents Support'
+
+    dependencies {
+        api "io.cloudevents:cloudevents-core:$cloudEventsVersion"
+        optionalApi "io.cloudevents:cloudevents-spring:$cloudEventsVersion"
+        optionalApi "io.cloudevents:cloudevents-json-jackson:$cloudEventsVersion"
+
+        optionalApi("io.cloudevents:cloudevents-avro-compact:$cloudEventsVersion") {
+            exclude group: 'org.apache.avro', module: 'avro'
+        }
+        optionalApi "org.apache.avro:avro:$avroVersion"
```

Comment:
I think we ca rely on whatever their module give us.
More over, I don't see a reason to have formats as compile dependencies at all.
Just `testImplementation`  should be enough for our use-case.
We really must not have any usage of their format implementations in our prod code.